wp-content/themes/inf_theme/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "inf_theme",
+  "version": "1.0.0",
+  "author": "Team Eightshift",
+  "private": true,
+  "main": "",
+  "scripts": {
+    "__eslintTheme": "eslint skin/assets/**/*.js",
+    "__stylelintTheme": "stylelint skin/assets/**/*.scss",
+    "lintStyle": "npm run __stylelintTheme",
+    "lintJs": "npm run __eslintTheme",
+    "lint": "npm run lintJs && npm run lintStyle && bash bin/lintPhp.sh",
+    "precommit": "npm run lint",
+    "start": "webpack --progress --watch --display-error-details --display-reasons --mode development",
+    "build": "NODE_ENV=production webpack --progress --mode production"
```

Comment:
I've received the feedback that `NODE_ENV` sometimes isn't working properly on windows, so one need to install [`win-node-env`](https://www.npmjs.com/package/win-node-env). We should document this, in wiki (maybe a FAQ section that we can link in the readme).